akka-actor-tests/src/test/scala/akka/AkkaVersionSpec.scala

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (C) 2009-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka
+
+import org.scalatest.{Matchers, WordSpec}
+
+class AkkaVersionSpec extends WordSpec with Matchers {
+
+  "The Akka version check" must {
+
+    "succeed if version is ok" in {
+      AkkaVersion.require("AkkaVersionSpec", Set("2.5.6"), "2.5.6")
+      AkkaVersion.require("AkkaVersionSpec", Set("2.5.6"), "2.5.7")
+      AkkaVersion.require("AkkaVersionSpec", Set("2.5.6", "2.4.19"), "2.4.19")
```

Comment:
when will we be able to use multiple versions like this? thinking about akka-http 10.0. how will we define/generate the required versions there? won't that come from the build dependency, and then we know nothing about 2.5.x?

Reply:
We'd have to manually put it in there, the version we actually depend on in the build ("2.4.19") we could probably get in but the 2.5.x version we require is either 2.5.0+ or some form of implicit knowledge, so that we would have to specify explicitly.

Reply:
Could be a synthetic use case though, maybe skip this and only support single-version to start with?

Reply:
I was thinking that if you really need it you could make two calls

Reply:
I think that wouldn't work as it would fail for one of them:
```
require("2.5.6")
require("2.4.19") 
```
There is no scenario where both are successful

Reply:
The reason I thought why we will need it is because I remembered when we added some classes/queryInterface2 types to persistence query, which would then be something like `Set("2.4.8", "2.5.0")`, but now that I think about it again, maybe that wasn't anyways possible because we didn't make those bincomp, and actually changed the type names.